### PR TITLE
Season 11 Mods

### PR DESCRIPTION
**Pull request by Jakosaur**
- Can leave this until they're in collections, otherwise it seems they'll show as collected.
- Sorted some strike loot to match collections.

### CODE REVIEW

**Comment by joshhunt** [comment]
nice!

**Comment by joshhunt** [comment]
Hey if you don't mind, could to take a look at sorting the categories within each energy group? Ill leave some comments on the code for an approach to do that

File: scripts/createModsData.js [hunk 1]
```diff
@@ -117,6 +117,13 @@ const matchArtifactMods = categoryName => ({ categoryName, matchArtifactMods });
...
+  matchPlugCategory('Seasonal Mod: Outlaw', 'enhancements.season_outlaw'),
+  matchPlugCategory('Seasonal Mod: Forge', 'enhancements.season_forge'),
+  matchPlugCategory('Seasonal Mod: Opulence', 'enhancements.season_opulence'),
+  matchPlugCategory('Seasonal Mod: Undying', 'enhancements.season_maverick'),
+  matchPlugCategory('Seasonal Mod: Dawn', 'enhancements.season_v470'),
+  matchPlugCategory('Seasonal Mod: Worthy', 'enhancements.season_v480'),
+  matchPlugCategory('Seasonal Mod: Arrivals', 'enhancements.season_v490'),
   // matchPlugCategory('General', 'enhancements.v2_general'), // must be before scavenger to prevent Circuit Scavenger mod from being classed as ammo scavenger
   matchName('Ammo Finder', /\sAmmo Finder$/g),
   matchName('Ammo Scavenger', /\sScavenger$/g),
```
Comment (joshhunt) [inline scripts/createModsData.js:129]: For each of these categories, put the name into a variable so we can refer to it later.

e.g.

```javascript
const CATEGORY_SEASONAL_ARTIFACT = "Seasonal Artifact mod";
const CATEGORY_SEASON_OUTLAW = "Seasonal Mod: Forge";
...etc
```

and then create a new array with the sort order:

```
const CATERGORY_DISPLAY_ORDER = [
 CATEGORY_GENERAL,
[basically all the mod categories],
[then the seasonal mods, in order]
]
```

the idea is that then each of the category names will be in their sorted order in the `CATERGORY_DISPLAY_ORDER` array

Comment (joshhunt) [reply scripts/createModsData.js:129]: and then down around line 250, add an extra sortBy at the end:

```javascript
          const groups = lodash(rawItems)
            .sortBy(item => item.index)
            .groupBy(item => categoriseMod(item, { artifactMods }))
            .mapValues(items => items.map(i => i.hash))
            .toPairs()
            .map(([name, items]) => ({ name, items }))
            .sortBy((group) => CATERGORY_DISPLAY_ORDER.indexOf(group.name))
            .value();
```

Comment (joshhunt) [reply scripts/createModsData.js:129]: I think that should be it to sort them in a sensible order?

Comment (joshhunt) [reply scripts/createModsData.js:129]: btw totally up to you if you want to do it :) don't feel pressured at all.

**Comment by Jakosaur** [comment]
All good! 🙂👍

Was there a reason behind reversing the icon opacity effect on the mods page compared to the main item sets? (collected/not collected) Nothing wrong with it, just somehow only just noticed the difference. 😂

**Comment by joshhunt** [comment]
Yeah, it just felt more appropriate for the mods page, at least at the time "knowing what mods i do have" made more sense. 🤷

**Comment by joshhunt** [comment]
sorting stuff is perfect - thanks for this!!!

**Pull request merged**